src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Components/Actions/EliminationForms/FailedToQualifyForm.razor

```diff
@@ -28,62 +29,41 @@
 </MudButton>
 
 @code {
-    [Inject]
-    IParticipationBehind _participationBehind { get; set; } = default!;
     [Parameter]
     public FailedToQualify? FailedToQualify { get; set; }
-    [Parameter]
-    public EventCallback OnRestore { get; set; }
-    [Parameter]
-    public EventCallback OnEliminate { get; set; }
 
-    private IEnumerable<string>? FTQReason { get; set; }
+    private IEnumerable<FTQCodes> Codes { get; set; } = new List<FTQCodes>();
```

Comment:
As this is a collection that is modified by this component it can be defined as a List directly